data/tabular/uniprot_sentences/meta.yaml

```diff
@@ -35,3 +35,10 @@ templates:
     - |-
       User: Describe the {#function|biological function!} of the {#protein|amino-acid sequence|AA sequence|polypeptide!} {other#}?
       Assistant: {sentences#}.
+    - |-
+      User: What {#protein|amino-acid sequence|AA sequence|polypeptide!} best fits the {#function|biological function!} described in the next sentence(s). {sentences#}
+      Assistant: The {#protein|amino-acid sequence|AA sequence|polypeptide!} that best fits the described function is {other#}.
+    - |-
+      Task: {#Generate|Create|Come up with!} a {AA sequence|protein} based on the description.
```

Comment:
```suggestion
      Task: {#Generate|Create|Come up with!} a {#AA sequence|protein!} based on the description.
```

Reply:
syntax was missing, please carry out a test run, then this should be immediately visible.